python/ray/dashboard/memory.py

```diff
@@ -0,0 +1,280 @@
+import base64
+import typing
+
+from collections import defaultdict
+from enum import Enum
+
+import ray
+
+
+def decode_object_id_if_needed(object_id: str) -> bytes:
```

Comment:
Do we know why these are sometimes base64 encoded and sometimes hex? That seems very strange. We should probably standardize instead of handling it here in a fairly hacky way (nbd if it's hard to track down).

Reply:
I tracked it down with some timebox (30 minutes) and could not find the reason. It seems to me that everything should be Base64 encoded because the prototype says this object id will be bytes types. I assume it could be some gPRC issues, but I am not sure. I will probably add another timeboxed task to figure this out.